MapleServer2/PacketHandlers/Game/RequestItemUseMultipleHandler.cs

```diff
@@ -73,17 +72,14 @@ public override void Handle(GameSession session, PacketReader packet)
                     {
                         if (index < item.Content.Count)
```

Comment:
Not a problem from your PR but this doesn't handle negative indices, should be fixed.